62.CAD/main.cpp

```diff
@@ -1635,12 +2712,12 @@ class CADApp : public ApplicationBase
 		logicalDevice->waitIdle();
 	}
 
-	double getScreenToWorldRatio(const float64_t3x3& viewProjectionMatrix, uint32_t2 windowSize)
+	float getScreenToWorldRatio(const float64_t3x3& viewProjectionMatrix, uint32_t2 windowSize)
```

Comment:
I suggest leaving this as float64_t (double) and cast it in the very last moment before storing it into struct, cause you sometimes need to invert this